recipe/meta.yaml

```
@@ -1,38 +1,37 @@
 {% set name = "deepchem" %}
-{% set version = "2.3.0" %}
+{% set version = "2.4.0" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/deepchem/deepchem/archive/{{ version }}.tar.gz
-  sha256: eec0d4dfa23e234d26d171ef20c62a699052a1bc230bbace341047e0c2673030
+  sha256: 25ebc59a53c8dcd841dbebab00bbef85efe8c29fa33e49c36b751cbd2f76110c
 
 build:
   noarch: python
-  number: 2
-  script: "PBR_VERSION=${PKG_VERSION} && export PBR_VERSION && {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv"
+  number: 0
+  script: PBR_VERSION=${PKG_VERSION} && export PBR_VERSION && {{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv
 
 requirements:
   host:
-    - python >=3.5
-    - pip
+    - python >=3.5,<3.8
+    - pip==20.2.*
```

```suggestion
    - pip
```
pip is not used for resolving dependencies, just for the installation of deepchem (see the script line above in the build section) so pinning here doesn't solve the problem